Polyhedron/demo/Polyhedron/Plugins/PMP/Scene_facegraph_item_k_ring_selection_config.h

```diff
@@ -1,7 +1,7 @@
 #ifndef SCENE_FACEGRAPH_ITEM_K_RING_SELECTION_CONFIG_H
 #define SCENE_FACEGRAPH_ITEM_K_RING_SELECTION_CONFIG_H
 
-#if defined(scene_poly_item_k_ring_selection_EXPORTS) || (scene_sm_item_k_ring_selection_EXPORTS)
+#if defined(scene_k_ring_selection_EXPORTS)
```

Comment:
I thought that kind of error was detected, on Linux, by `-fvisibility=hidden` and `-z defs`. That does not seem to work.